src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -198,34 +231,36 @@
 
     private final BufferPool bufferPool;
     private final BudgetCreditor creditor;
+    private final LongFunction<BudgetDebitor> supplyDebitor;
 
     private final AmqpServerDecoder decodeFrameType = this::decodeFrameType;
     private final AmqpServerDecoder decodeHeader = this::decodeHeader;
     private final AmqpServerDecoder decodeOpen = this::decodeOpen;
     private final AmqpServerDecoder decodeBegin = this::decodeBegin;
     private final AmqpServerDecoder decodeAttach = this::decodeAttach;
     private final AmqpServerDecoder decodeFlow = this::decodeFlow;
+    private final AmqpServerDecoder decodeTransfer = this::decodeTransfer;
     private final AmqpServerDecoder decodeClose = this::decodeClose;
     private final AmqpServerDecoder decodeIgnoreAll = this::decodeIgnoreAll;
     private final AmqpServerDecoder decodeUnknownType = this::decodeUnknownType;
 
     private final int outgoingWindow;
     private final String containerId;
-    private final long defaultMaxFrameSize;
+    private final long decodeMaxFrameSize;
```

Comment:
this field should be called `maxFrameSize` to match the configuration setting
`decodeMaxFrameSize` should be a member field on the `AmqpServer`, initialized to `MIN_MAX_FRAME_SIZE`.